surfer/viz.py

```diff
@@ -960,8 +960,9 @@ def add_overlay(self, source, min=2, max="robust_max", sign="abs",
         self.overlays_dict[name] = ol
         self._toggle_render(True, views)
 
-    def add_data(self, array, min=None, max=None, thresh=None,
-                 colormap="RdBu_r", alpha=1,
+    def add_data(self, array, min=None, max=None, mid=None, 
+                 thresh=None, center=None, transparent=False, 
```

Comment:
even though it's annoying, we probably need to put `mid`, `center`, and `transparent` at the end of the argument list for backward compatibility